static/app/views/dashboardsV2/releasesSelectControl.tsx

```diff
@@ -88,7 +88,20 @@ function ReleasesSelectControl({
       onChange={opts => setActiveReleases(opts.map(opt => opt.value))}
       onClose={() => {
         resetSearch();
-        handleChangeFilter?.({[DashboardFilterKeys.RELEASE]: activeReleases});
+        const activeReleasesVersions = new Set(activeReleases);
+
+        const activeReleasesById = releases
+          .filter(release => activeReleasesVersions.has(release.version))
+          .map(release => release.id);
+
+        if (activeReleasesVersions.has('latest')) {
+          activeReleasesById.push('latest');
+        }
```

Comment:
If we're going to use these ids to pull the metadata for releases, do these two lists need to be in sync in terms of the order?

For example, if I select releases in the following order `abc@1.0.0, latest, abc@1.1.0`, 
`activeReleases` will have this order [`abc@1.0.0`, `latest`, `abc@1.1.0`]
but `activeReleasesById` will have this order [1, 2, latest] since we push that in at the end. Not sure if that will cause a problem 👀